l2geth/eth/api_backend.go

```diff
@@ -282,7 +264,7 @@ func (b *EthAPIBackend) GetEVM(ctx context.Context, msg core.Message, state *sta
 	vmError := func() error { return nil }
 
```

Comment:
Need to add:

https://github.com/ethereum/go-ethereum/blob/64c53edf836efb1d55b85e841db364a424e95422/eth/api_backend.go#L206-L208